src/EFTools/DesignXmlCore/VirtualTreeGrid/DropDownControl.cs

```diff
@@ -2229,10 +2229,11 @@ public void DoModalLoop()
             // Push a modal loop.  This kind of stinks, but I think it is a
             // better user model than returning from DropDownControl immediately.
             //  
+            IntPtr[] emptyHandles = new IntPtr[0];
```

Comment:
The Win32 docs don't explicitly state anywhere I could find that passing nCount = 0 allows you to pass a null array, so this is making it explicit.

Reply:
nit: Array.Empty<IntPtr>()? I think it then caches the empty array so you only ever make one

Reply:
Ah, right, handy.